docs/development/use-bundled-plugins.md

```diff
@@ -0,0 +1,63 @@
+# Using bundled plugins in local development
+
+## You want a quick way to add the security plugin to your local env
+
+### 1. Start an instance of OpenSearch that already includes the security plugin
+
+By default, we generally develop against a "minimal" version of OpenSearch, with no external plugins installed (such as when [running `yarn opensearch snapshot`](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/DEVELOPER_GUIDE.md#run-opensearch)). This is great to make sure we understand the basic behavior of the core app. But that's not how most users install or run OpenSearch - they generally use all the bundled "default" plugins, of which [`security`](https://github.com/opensearch-project/security) is probably the most important.
+
+#### Install and run a "full" version of OpenSearch via tarball
+
+_(Do this on the same machine where you're developing on OpenSearch Dashboards - these examples assume you're using [an Ubuntu dev environment](https://quip-amazon.com/umanA1aLbFsw/Setup-Ubuntu-Machine-for-OpenSearch-devtest))_
+
+1. Make/navigate to a directory where you want to store/run OpenSearch tarballs
+2. Download the latest version of the OpenSearch tarball
```

Comment:
We can also link them to the opensearch.org [downloads page](https://opensearch.org/downloads.html#opensearch). Right clicking on the download button and using that as the url to curl is a useful trick if you dont want to create the URL manually